python/hackathon_app/tests/data/data.json

```diff
@@ -688,4 +728,4 @@
     "properties": {
         "title": "Hackathons Test Data"
     }
-}
+}
```

Comment:
I think your editor is failing to add a new line at end of file. might be a configuration thing